### DISCUSSION TITLE
Test Data for different SOP Classes

### DISCUSSION THREAD

**Discussion by Vishal-Tahilramani** [fo-dicom/fo-dicom#1711] [General]
Hi Team,

One of my client's requirement is to test all set of SOP Classes in DICOM and I am looking for some test data repository to have some sample annonymized DICOM data sets/files for all SOP Classes in the list below.

 
1.2.840.10008.1.2	    Implicit VR Endian: Default Transfer Syntax for DICOM	 
1.2.840.10008.1.2.1	    Explicit VR Little Endian	 
1.2.840.10008.1.2.1.99  Deflated Explicit VR Little Endian	 
1.2.840.10008.1.2.2	    Explicit VR Big Endian	 
 
1.2.840.10008.1.2.4.50	JPEG Baseline (Process 1): Default Transfer Syntax for Lossy JPEG 8-bit Image Compression	 
1.2.840.10008.1.2.4.51	JPEG Baseline (Processes 2 & 4): Default Transfer Syntax for Lossy JPEG 12-bit Image Compression (Process 4 only)	 
1.2.840.10008.1.2.4.52	JPEG Extended (Processes 3 & 5)	Retired
1.2.840.10008.1.2.4.53	JPEG Spectral Selection, Nonhierarchical (Processes 6 & 8 ) Retired
1.2.840.10008.1.2.4.54	JPEG Spectral Selection, Nonhierarchical (Processes 7 & 9)	Retired
1.2.840.10008.1.2.4.55	JPEG Full Progression, Nonhierarchical (Processes 10 & 12)	Retired
1.2.840.10008.1.2.4.56	JPEG Full Progression, Nonhierarchical (Processes 11 & 13)	Retired
1.2.840.10008.1.2.4.57	JPEG Lossless, Nonhierarchical (Processes 14)	 
1.2.840.10008.1.2.4.58	JPEG Lossless, Nonhierarchical (Processes 15)	Retired
1.2.840.10008.1.2.4.59	JPEG Extended, Hierarchical (Processes 16 & 18)	Retired
1.2.840.10008.1.2.4.60	JPEG Extended, Hierarchical (Processes 17 & 19)	Retired
1.2.840.10008.1.2.4.61	JPEG Spectral Selection, Hierarchical (Processes 20 & 22)	Retired
1.2.840.10008.1.2.4.62	JPEG Spectral Selection, Hierarchical (Processes 21 & 23)	Retired
1.2.840.10008.1.2.4.63	JPEG Full Progression, Hierarchical (Processes 24 & 26)	Retired
1.2.840.10008.1.2.4.64	JPEG Full Progression, Hierarchical (Processes 25 & 27)	Retired
1.2.840.10008.1.2.4.65	JPEG Lossless, Nonhierarchical (Process 28)	Retired
1.2.840.10008.1.2.4.66	JPEG Lossless, Nonhierarchical (Process 29)	Retired
1.2.840.10008.1.2.4.70	JPEG Lossless, Nonhierarchical, First- Order Prediction (Processes 14 [Selection Value 1]): Default Transfer Syntax for Lossless JPEG Image Compression	 
1.2.840.10008.1.2.4.80	JPEG-LS Lossless Image Compression	 
1.2.840.10008.1.2.4.81	JPEG-LS Lossy (Near- Lossless) Image Compression	 
1.2.840.10008.1.2.4.90	JPEG 2000 Image Compression (Lossless Only)	 
1.2.840.10008.1.2.4.91	JPEG 2000 Image Compression	 
1.2.840.10008.1.2.4.92	JPEG 2000 Part 2 Multicomponent Image Compression (Lossless Only)
1.2.840.10008.1.2.4.93	JPEG 2000 Part 2 Multicomponent Image Compression	 
 
1.2.840.10008.1.2.4.94	JPIP Referenced	 
1.2.840.10008.1.2.4.95	JPIP Referenced Deflate	 
1.2.840.10008.1.2.5	        RLE Lossless	 
1.2.840.10008.1.2.6.1	RFC 2557 MIME Encapsulation	 
 
1.2.840.10008.1.2.4.100	MPEG2 Main Profile Main Level	 
1.2.840.10008.1.2.4.102	MPEG-4 AVC/H.264 High Profile / Level 4.1	 
1.2.840.10008.1.2.4.103	MPEG-4 AVC/H.264 BD-compatible High Profile / Level 4.1

Can some one point me to sample test data repository for above set of SOP Classes? 

Thanks,
Vishal T

**Comment by mrbean-bremen** [comment]
There is no ready collection for this as far as I know (though some efforts exist to make one), but I would check [this site](https://www.aliza-dicom-viewer.com/download/datasets) for a start. Also check the links to other datasets down at that site ("Sites with data sets collections").
